core/consensus/grandpa/impl/grandpa_impl.cpp

```diff
@@ -790,21 +789,17 @@ namespace kagome::consensus::grandpa {
                 block_info);
         return VotingRoundError::NO_KNOWN_AUTHORITIES_FOR_BLOCK;
       }
-      auto &authorities = authorities_opt.value();
+      auto &authority_set = authorities_opt.value();
 
       // This is justification for non-actual round
-      if (authorities->id < current_round_->voterSetId()
-          or (authorities->id == current_round_->voterSetId()
+      if (authority_set->id < current_round_->voterSetId()
+          or (authority_set->id == current_round_->voterSetId()
               && justification.round_number < current_round_->roundNumber())) {
         return VotingRoundError::JUSTIFICATION_FOR_ROUND_IN_PAST;
       }
 
-      if (authorities->id > current_round_->voterSetId() + 1) {
-        return VotingRoundError::WRONG_ORDER_OF_VOTER_SET_ID;
```

Comment:
Why was check of order removed?

Reply:
I had a suspicion it fails on forced change, but I'll re-check.

Reply:
Yeah, it's okay, the problem was elsewhere.